typescript/web/src/components/workspace-switcher/workspace-menu/workspace-selection-popover/workspace-selection-popover.tsx

```diff
@@ -143,81 +144,95 @@ export const WorkspaceSelectionPopover = ({
       initialFocusRef={searchInputRef}
     >
       <PopoverTrigger>{trigger}</PopoverTrigger>
-      <PopoverContent
-        borderColor={borderColor}
-        cursor="default"
-        pointerEvents="initial"
-        aria-label={ariaLabel}
-      >
-        <PopoverBody pl="0" pr="0" pt="0">
-          <Box>
-            <Box {...getComboboxProps()} pl="3" pr="3" pt="3">
-              <InputGroup>
-                <InputLeftElement pointerEvents="none">
-                  <MagnifierIcon fontSize="2xl" />
-                </InputLeftElement>
-                {/* Visually hidden accessible label. See: https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text */}
-                <Text
-                  as="label"
-                  {...getLabelProps()}
-                  border={0}
-                  clip="rect(0 0 0 0)"
-                  height="1px"
-                  margin="-1px"
-                  overflow="hidden"
-                  padding={0}
-                  position="absolute"
-                  width="1px"
-                >
-                  Search in workspace selection popover
-                </Text>
-                <Input
-                  {...getInputProps({ ref: searchInputRef })}
-                  name="workspace-selection-search"
-                  placeholder="Search..."
-                  pr="4rem"
-                />
-                <InputRightElement
-                  width="4rem"
-                  justifyContent="flex-end"
-                  pr="2"
-                >
-                  {inputValue && (
-                    <>
-                      <CloseCircleIcon
-                        fontSize="2xl"
-                        onClick={reset}
-                        cursor="pointer"
-                        color={closeCircleIconColor}
+      <Portal>
```

Comment:
Because there was a property set in the Breadcrumb component to hide the overflow on the X direction. This property was inherited by all its children and notably the workspace selection popover, and this was causing Cypress to think the popover was not visible when actually it was. So rendering it in a portal enables to avoid inheriting that prop on the popover and Cypress does not complain anymore